code-reports/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No jacoco can only aggregate modules built before it in the reactor so can't be in the parent.